lib/Sema/PreCheckExpr.cpp

```diff
@@ -427,7 +453,8 @@ Expr *TypeChecker::resolveDeclRefExpr(UnresolvedDeclRefExpr *UDRE,
     // e.g. (x*-4) that needs whitespace.
     if (diagnoseRangeOperatorMisspell(Context.Diags, UDRE) ||
         diagnoseIncDecOperator(Context.Diags, UDRE) ||
```

Comment:
See this line for an example of the name of a function that diagnoses a specific operator.